### PR TITLE
Tests cleanup

### PR DESCRIPTION
**Pull request by jdom**
Cleanup of some tests, in preparation to the upcoming TestingSiloHost change.

### CODE REVIEW

**Comment by jdom** [comment]
@dotnet-bot test this please

**Comment by jdom** [comment]
@sergeybykov tests already passed, so feel free to merge asap after review. I thought it didn't run on the last commit so I re-queued, but it actually run correctly. It's just that because I reordered some commits, GitHub shows them in the wrong (original) order.

File: test/Tester/StreamingTests/StreamFilteringTests.cs [hunk 1]
```diff
@@ -249,155 +228,143 @@ public virtual void Dispose()
...
+        [Fact, TestCategory("Functional"), TestCategory("Streaming"), TestCategory("Filters"), TestCategory("Azure")]
         public async Task AQ_Filter_EvenOdd()
         {
-            streamProviderName = StreamTestsConstants.AZURE_QUEUE_STREAM_PROVIDER_NAME;
-
             await Test_Filter_EvenOdd();
         }
 
-        [Fact(Skip = "Ignored"), TestCategory("Functional"), TestCategory("Streaming"), TestCategory("Filters"), TestCategory("Azure")]
+        [Fact, TestCategory("Functional"), TestCategory("Streaming"), TestCategory("Filters"), TestCategory("Azure")]
```
Comment (sergeybykov) [inline test/Tester/StreamingTests/StreamFilteringTests.cs]: Is it passing now, so we don't skip it anymore?

Comment (jdom) [inline test/Tester/StreamingTests/StreamFilteringTests.cs]: yes, they are passing now

**Pull request merged**